src/services/monitoring.ts

```diff
@@ -1,10 +1,163 @@
+    },
+    include: [
+      {
+        model: GrantNumberLink,
+        as: 'grantNumberLink',
+        required: true,
+        include: [
+          {
+            model: MonitoringReviewGrantee,
+            attributes: ['id', 'grantNumber', 'reviewId'],
+            required: true,
+            as: 'monitoringReviewGrantees',
+            include: [
+              {
+                model: MonitoringReviewLink,
+                as: 'monitoringReviewLink',
+                required: true,
+                include: [
+                  {
+                    model: MonitoringReview,
+                    as: 'monitoringReviews',
+                    attributes: [
+                      'reportDeliveryDate',
+                      'id',
+                      'reviewType',
+                      'reviewId',
+                      'statusId',
+                    ],
+                    required: true,
+                    include: [
+                      {
+                        attributes: ['id', 'statusId'],
+                        model: MonitoringReviewStatusLink,
+                        as: 'statusLink',
+                        required: true,
+                        include: [
+                          {
+                            attributes: ['id', 'name', 'statusId'],
+                            model: MonitoringReviewStatus,
+                            as: 'monitoringReviewStatuses',
+                            required: true,
+                          },
+                        ],
+                      },
+                    ],
+                  },
+                ],
+              },
+            ],
+          },
+        ],
+      },
+    ],
+  });
+
+  // get the first grant (remember, there can only be one)
+  const grant = (grants[0]?.toJSON() || null);
+
+  if (!grant) {
+    // not an error, it's valid for there to be no findings for a grant
+    return null;
+  }
+
+  // since all the joins made in the query above are inner joins
+  // we can count on the rest of this data being present
+  const { monitoringReviewGrantees } = grant.grantNumberLink;
+
+  // get the most recent review
+  // - 1) first extract from the join tables
+  const monitoringReviews = monitoringReviewGrantees.map(
+    (review: IMonitoringReviewGrantee) => review.monitoringReviewLink.monitoringReviews,
+  ).flat();
+
+  // - 2) then sort to get the most recent
+  const monitoringReview = monitoringReviews.reduce((
+    a: IMonitoringReview,
+    b: IMonitoringReview,
+  ) => {
+    if (a.reportDeliveryDate > b.reportDeliveryDate) {
+      return a;
+    }
+    return b;
+  }, monitoringReviews[0]);
+
+  // from the most recent review, get the status via the statusLink
+  const { monitoringReviewStatuses } = monitoringReview.statusLink;
+
+  // I am presuming there can only be one status linked to a review
+  // as that was the structure before tables were refactored
+  const [status] = monitoringReviewStatuses;
+
```

Comment:
@hardwarehuman or @GarrettEHill I think you would be the best ones to confirm my assumption on lines 150-152?